### PR TITLE
create engine service proto definition and regenerate bindings script

### PR DESCRIPTION
**Pull request by leoporoli**

### CODE REVIEW

File: kurtosis-engine-rpc-api/engine_service.proto [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+message CreateEnclaveResponse {
+  //The Docker network id that is part of the Kurtosis Enclave
+  string network_id = 1;
+  //The Docker network IP
+  string network_ip = 2;
+  //The API container ID that is currently running in the Kurtosis Enclave
+  string api_container_id = 3;
+  //The API container IP address
+  string api_container_ip = 4;
+  //THE API container Host IP address
```
Comment (mieubrisse) [inline kurtosis-engine-rpc-api/engine_service.proto:41]: Recomment comment update to `The IP of the interface on the host machine where the API container's port can be accessed`

File: kurtosis-engine-rpc-api/engine_service.proto [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  string api_container_host_ip = 5;
+  //THE API container Host port number
+  string api_container_host_port = 6;
+}
+
+// ==============================================================================================
+//                                          Get Enclave
+// ==============================================================================================
+message GetEnclaveArgs {
+  //The ID of the new Kurtosis Enclave
```
Comment (mieubrisse) [inline kurtosis-engine-rpc-api/engine_service.proto:51]: Gotta make sure to update comments when copy-pasting; this should be `The ID of the enclave to retrieve`

File: kurtosis-engine-rpc-api/engine_service.proto [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+// ==============================================================================================
+//                                       Destroy Enclave
+// ==============================================================================================
+message DestroyEnclaveArgs {
+  //The ID of the new Kurtosis Enclave
+  string enclave_id = 1;
+  //The Docker network id that is part of the Kurtosis Enclave
+  string network_id = 2;
```
Comment (mieubrisse) [inline kurtosis-engine-rpc-api/engine_service.proto:78]: Neither `network_id` nor `api_container_id` should be needed here; the Engine can look these values up using the enclave ID & labels API

File: typescript/package.json [hunk 4]
```diff
@@ -1,22 +1,42 @@
...
     ],
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/kurtosis-tech/kurtosis-engine-api-lib.git"
+    },
+    "keywords": [
+        "kurtosis",
+        "engine"
+    ],
+    "author": "Kurtosis Technologies Inc <accounts@kurtosistech.com> (https://www.kurtosistech.com/)",
```
Comment (mieubrisse) [inline typescript/package.json:22]: Oo `accounts@` is actually the email we use to sign up for services (e.g. Quip); you'll want `support@kurtosistech.com` instead

File: typescript/package.json [hunk 5]
```diff
@@ -1,22 +1,42 @@
 {
-    "//": "TODO TODO Add package name, description, license, etc. here",
+    "name": "kurtosis-engine-api-lib",
     "version": "0.1.0",
```
[the remark this answers is not in the document]
Comment (leoporoli) [reply typescript/package.json:3]: Yes, there is

**Pull request merged**